app/views/tracks/exercises/show/_action_box_dig_deeper_practice_exercise.html.haml

```diff
@@ -0,0 +1,23 @@
+%p.mb-16
+  Solving the exercise is just the first step to improving. It's now time to learn from others. These steps are optional but will vastly enhance your knowledge.
+
+= render("tracks/exercises/show/action_box_dig_deeper_mentoring", track:, exercise:, solution:)
+
+- if user_track.viewed_community_solutions.joins(:solution).where.not(solution:).where(solution: { exercise: }).count >= 5
```

Comment:
Suggest moving this into `user_track.num_viewed_community_solutions(exercise)` or something?